webu.c

```diff
@@ -1391,6 +1392,9 @@ static void webu_html_script_action(struct context **cnt, struct webui_ctx *webu
         "      document.getElementById('act_btn').style.display=\"none\"; \n"
         "      document.getElementById('cfg_value').value = '';\n"
         "      document.getElementById('cfg_parms').value = 'default';\n"
+        "      if ((actval == \"/detection/pause\") || (actval == \"/detection/start\")) {\n"
+        "        window.location.reload(); "
```

Comment:
Firefox (possibly more) seem to dislike this and the action doesn't get sent to motion correctly/at all (ie. pause and start aren't pausing or starting). Verified this by commenting these 3 lines out and re-trying, with pause/start actually working again. This is probably the same or a quite similar issue to what I had with trying to close the menu after click from the JS, with the action failing to happen then.

Reply:
I tried with a timed reload (500ms) and it worked: `setTimeout(function(){ window.location.reload(); }, 500);`  
Not the **best** solution but better than no response.

Reply:
I pushed a commit to add a listener event to this and it seems to be more reliable.  I forgot these are async requests.

Reply:
Ah got ya; works good now.